portfolio/pom.xml

```diff
@@ -23,6 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <!-- Add Gson library to the project -->
```

Comment:
this comment can be removed since that's quite obvious.

Reply:
based on my personal industry experience, the comments in pom file can be used for explaining like why specific version is used.

Reply:
Ahh okay, I'll keep that mind then and remove the comments. I'm not exactly sure why we use the versions that we use though, it's just what we were given.